packages/main/src/plugin/filesystem-monitoring.ts

```diff
@@ -34,6 +53,10 @@ export class FileSystemWatcherImpl implements containerDesktopAPI.FileSystemWatc
       const uri: containerDesktopAPI.Uri = Uri.file(addedPath);
       this._onDidCreate.fire(uri);
     });
+    watcher.on('addDir', (addedPath: string) => {
+      const uri: containerDesktopAPI.Uri = Uri.file(addedPath);
+      this._onDidCreate.fire(uri);
+    });
```

Comment:
I don't know why the `addDir` event was not listened. If it is on purpose, I will need to look at another way.